docs/team/junxiang.md

```diff
@@ -9,28 +9,59 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
+- Implemented the functionality to create tag, delete tag, as well as tag or untag taggable objects.
+- Implemented `ParserManager` class which manages the creation of different parser functions to parse the user input to the required parameters.
+- Implemented `SystemException` class which extends from `Exception` class.
+    - Within it, there is an enumeration for all possible types of exception that may occur and each enum value contains a unique string as its message.
+- Implemented `Formatter` class which formats the different objects, such as Note, Event and String with a specified layout into a String to be printed.
     
 ### Code contributed
+[RepoSense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=chongjx&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+
+### Project management
+- Managed releases `v1.0` and `v2.0` on GitHub
 
 ### Contributions to User Guide
-1.
-1. 
+Added documentation for following features:
+
+1. [3.11 Create Tags](https://ay2021s1-cs2113-t13-1.github.io/tp/UserGuide.html#create-t)
+1. [3.12 List Tags](https://ay2021s1-cs2113-t13-1.github.io/tp/UserGuide.html#list-t)
+1. [3.13 Tag/Untag Notes](https://ay2021s1-cs2113-t13-1.github.io/tp/UserGuide.html#tag-n)
+1. [3.14 Tag/Untag Events](https://ay2021s1-cs2113-t13-1.github.io/tp/UserGuide.html#tag-e)
+1. [3.15 Delete Tags](https://ay2021s1-cs2113-t13-1.github.io/tp/UserGuide.html#delete-t)
 
 ### Contributions to Developer Guide
-1. Setting up
-1. Updated architecture overview
+Added implementation details for the following:
+
+1. [2.1 Architecture Overview](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#overview)
+1. [2.3 Parser & ParserManager](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#parserManager)
+1. [2.4 Commands`](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#commands)
+    - Class diagram of Command Class, Figure 6
+1. [2.7 Tags`](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#tag)
+1. [2.9 User Interface](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#ui)
+1. [2.10 SystemException](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#exception)
+1. [2.11 Usage of External Libraries](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#color)
+1. [8.1 List of ParseCommand Classes](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#parseXYZCommands)
+1. [8.2 List of Command Classes](https://ay2021s1-cs2113-t13-1.github.io/tp/DeveloperGuide.html#XYZCommands)
 
 ### Community
-(https://nus-cs2113-ay2021s1.github.io/addressbook-level3/team/johndoe.html)
+1. Reviewed most of the PRs to ensure the Classes and functions follow the project architecture as well code logic of the programme.
+    - Reviewed a total of 73 PRs out of 114 PRs, with 167 comments, dated 7/11/2020.
```

Comment:
If you are using the GitHub stats for PRs reviewed by you. Some of these PRs are authored by you (because of your responses) so you might want to double-check and do a manual count.

Reply:
Noted thanks!